services/movie_session.py

```diff
@@ -0,0 +1,60 @@
+from datetime import datetime
+
+from typing import Optional
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=movie,
+        cinema_hall=cinema_hall
+    )
```

Comment:
All tests should pass when you write the code in the following form without using direct objects on lines 15 and 16
```suggestion
    MovieSession.objects.create(
        show_time=movie_show_time,
        movie_id=movie_id,
        cinema_hall_id=cinema_hall_id
    )
```
In case they do not pass locally, please, visit a Q&A to understand the reason. Adjust the code to create movie session using ids of cinema hall and movie